### PR TITLE
[Package Add] Add batsignal

### PR DESCRIPTION
**Pull request by harsh18262**
This PR adds `batsignal`

### CODE REVIEW

File: packages/batsignal/batsignal.pacscript [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+description="A lightweight battery monitor daemon"
+hash="c9e17c1e02355d257952d13072791063240b42ebfb4e0f30cc28fe944aa6de4d"
+maintainer="Harshwardhan Mehrotra <harshwardhanmehrotra@gmail.com>"
+
+prepare() {
+        true
+}
+
+build() {
+	make -j$(nproc)C
```
Comment (wizard-28) [inline packages/batsignal/batsignal.pacscript:15]:
```suggestion
	make -j$(nproc)
```

Rouge `C`

**Comment by NarutoXY** [comment]
You also have to edit `packagelist`

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/batsignal/batsignal.pacscript (hunk 1 above)
Comment (harsh18262) [reply packages/batsignal/batsignal.pacscript]: Thanks

**Comment by wizard-28** [comment]
It works in my machine but the CI fails
```bash
~
❯ pacstall -Il batsignal
Do you want to view the pacscript first [y/N]
Do you want to edit the pacscript [y/N]
[+] INFO: Sourcing pacscript
[+] INFO: URL exists
[+] INFO: batsignal requires libnotify-dev make pkg-config gcc to install
Do you want to remove them after installing batsignal [y/N]
Selecting previously unselected package libgdk-pixbuf2.0-dev:amd64.
(Reading database ... 178074 files and directories currently installed.)
Preparing to unpack .../libgdk-pixbuf2.0-dev_2.40.0+dfsg-3ubuntu0.2_amd64.deb ...
Unpacking libgdk-pixbuf2.0-dev:amd64 (2.40.0+dfsg-3ubuntu0.2) ...
Selecting previously unselected package libnotify-dev:amd64.
Preparing to unpack .../libnotify-dev_0.7.9-1ubuntu2_amd64.deb ...
Unpacking libnotify-dev:amd64 (0.7.9-1ubuntu2) ...
Setting up libgdk-pixbuf2.0-dev:amd64 (2.40.0+dfsg-3ubuntu0.2) ...
Setting up libnotify-dev:amd64 (0.7.9-1ubuntu2) ...
[+] INFO: Installing dependencies
[+] INFO: Retrieving packages
[+] INFO:
Initializing download: https://github.com/electrickite/batsignal/archive/refs/tags/1.2.0.zip
File size: unavailable
Opening output file 1.2.0.zip
Server unsupported, starting from scratch with one connection.
Starting download

Connection 0 finished

Downloaded 9.37012 Kilobyte(s) in 1 second(s). (7.79 KB/s)
c99 -pedantic -Wall -Wextra -Werror -Wno-unused-parameter -Os -s -O -std=c99 -pthread -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include   -c -o batsignal.o batsignal.c
sed "s/VERSION/1.2.0/g" < batsignal.1.in > batsignal.1
c99 -o batsignal batsignal.o -lnotify -lgdk_pixbuf-2.0 -lgio-2.0 -lgobject-2.0 -lglib-2.0
[+] INFO: Installing
Installing in /usr/src/pacstall/batsignal/usr/local
install -Dm755 -t /usr/src/pacstall/batsignal/usr/local/bin batsignal
install -Dm644 -t /usr/src/pacstall/batsignal/usr/local/share/man/man1 batsignal.1
[+] INFO: Symlinking files
[+] INFO: Storing pacscript
[+] INFO: Cleaning up
```

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**